vignettes/fims-setup-guide.Rmd

```diff
@@ -0,0 +1,177 @@
+
+```{r common-setup, echo=FALSE}
+# The bash script for setting up FIMS
+common_script <- "bash <(curl -sL https://raw.githubusercontent.com/NOAA-FIMS/FIMS/main/setup_fims.sh)"
+
+# Fetch the remote R script for the data bucket example
+readwrite_databucket_url <- "https://raw.githubusercontent.com/nmfs-opensci/CloudComputingSetup/refs/heads/main/R/readwrite_databucket.R"
+readwrite_databucket_script <- tryCatch(
+  readLines(readwrite_databucket_url, warn = FALSE),
+  error = function(e) "# Could not load remote script."
+)
+
+# Fetch the remote terminal script for mounting a data bucket
+mount_bucket_url <- "https://raw.githubusercontent.com/nmfs-opensci/CloudComputingSetup/refs/heads/main/R/mount_bucket.R"
+mount_bucket_script <- tryCatch(
+  readLines(mount_bucket_url, warn = FALSE),
+  error = function(e) "# Could not load remote script."
+)
+```
+
+## ☁️ Google Cloud Workstations
+
+NOAA Fisheries Google Cloud Workstations provide a managed RStudio, Visual Studio Code (VS Code), or Positron environment. For detailed visual guides (e.g., snapshots) on starting, stopping, or managing workstations, please refer to the [Internal NOAA documentation](https://docs.google.com/document/d/1nziPdPULoRWOYQ9WKzISNUgJvANACvfYpFr1z3Ro2Bc/edit?tab=t.0#heading=h.5509095bxzv7).
+
+- Navigate to the NOAA Fisheries [Google Cloud Workstations Console](https://console.cloud.google.com/workstations/list?project=ggn-nmfs-wsent-prod-1)  and click `+ create workstation`.
+
+- Provide a name for the workstation, select a configuration, and click the `Create` button.
+
+> **Note:** Use `posit-medium` or `posit-large` for workshops and rapid testing. It has faster startup times, though it carries higher cost due the Posit licensing fees. Select `nmfs-rstudio-medium` or `nmfs-rstudio-large` for long-term use as they avoid third-party licensing cost. Configurations (`*-small`) with low core counts will lead to failed compilation of FIMS.
+  
+- Click the `Start` and `Launch` buttons to initialize the workstation.
+
+- Add a new session from either `Positron Pro`, `RStudio Pro`, or `VS Code` when using `posit-*` configurations, and then `Launch` the session.
+
+- Open a bash terminal within your session and execute the following command to install all necessary system and R dependencies.
+
+```{r, echo=FALSE, results='asis'}
+cat("```bash\n")
+cat(common_script, sep = "\n")
+cat("\n```")
+```
+
+> **Note:** The process typically takes 15 to 30 minutes. If using `Positron Pro`, a restart of the R session is required to initialize the new libraries. Please run `library(FIMS)` in the R session to confirm the successful installation of FIMS.
```

Comment:
We should include what output the user should expect to see when they run `library(FIMS)`. I think none if it was successful and an error if it was not?